### DISCUSSION TITLE
Honeygain Windows version

### DISCUSSION THREAD

**Discussion by Vincent964** [engageub/InternetIncome#159] [Q&A]
hello congratulations for this project, for a while now I've been trying to run honeygain windows version on docker linux containers. Because I think that the windows version would allow me to have more earnings, do you know a way on how I could succeed? I've tried many times with wine but it doesn't always work and most of the time I always get errors. Thanks

**Comment by engageub** [comment]
Hello,
Thank you for your feedback. 
If you wish to use wine inside docker container there may be some dependencies missing. 
Also, there might be compatibility issues if you are using arm64 instead of amd64 architecture. 
Login is manual as Honeygain Windows App Requires you to input the credentials inside the GUI. If there is a sequence of keyboard commands which can lead to Honeygain GUI and its inputs, it can be automated. 
If you are able to succeed through the regular linux terminal using Wine, then you can create a docker image.

For issues related to Wine, you can contact them through https://forum.winehq.org/ or https://www.winehq.org/help

You can also try [CrossOver](https://www.codeweavers.com/store?ad=1034) which lets you run windows apps on linux machines.
If you are able to run this with crossOver, then you can create docker image with crossOver built inside the container. 


Thank you

**Reply by Vincent964** [reply]
Hi, I managed to start honeygain with wine 9.20 and enter the login credentials but I only recognized a connection problem, in particular by running wireshark very often you notice how the TCP connection resets. I posted a request for help in the wine forum as you advised me, where I uploaded the entire log but no one answered me, I also tried with the assistance of crossover but nothing. Please help me

https://forum.winehq.org/viewtopic.php?t=39608

**Reply by engageub** [reply]
Hello,
Wine support may take time to respond.
Have you contacted code-weavers. using the following link. 
https://www.codeweavers.com/contact-us



> in particular by running wireshark very often you notice how the TCP connection resets

From the above response, I did not understand what you mean by TCP connection resets.  Are you using direct connection or proxy connection. Do you mean the connection disconnects after logging into honeygain. 
I have not tried honeygain windows version on linux, so I cannot suggest you further on this.  If there is any issue you will have to contact the developers of winehq or codeweavers. 


Thank you
